config/makeWarp.py

```diff
@@ -0,0 +1,11 @@
+import os.path
+
+# Load configs shared between makeWarp and assembleCoadd
+config.load(os.path.join(os.path.dirname(__file__), "coaddBase.py"))
+
+config.doApplySkyCorr = False
```

Comment:
Please add a short explanatory comment for where this and subsequent config overrides originate from (e.g., matching obs_subaru for ease of 1:1 comparisons, or something else?)